frontend/amundsen_application/static/js/pages/SearchPage/SearchFilter/CheckBoxFilter/index.tsx

```diff
@@ -100,27 +90,21 @@ export class CheckBoxFilter extends React.Component<CheckBoxFilterProps> {
 
 export const mapStateToProps = (state: GlobalState, ownProps: OwnProps) => {
   const filterState = state.search.filters;
-  let filterValues = filterState[state.search.resource]
+  const filterValues = filterState[state.search.resource]
     ? filterState[state.search.resource][ownProps.categoryId]
-    : {};
-  if (!filterValues) {
-    filterValues = {};
-  }
-
+    : undefined;
+  const value = filterValues ? filterValues.value : '';
   return {
-    checkedValues: filterValues,
+    checkedValues: value ? value.split(',') : [],
```

Comment:
Nit: I would create a `const checkedValues = value ? value.split(',') : []` above and make this line just `checkedValues,`